Update conditional-access-app-control-how-to-overview.md (#745)
MicrosoftDocs/CloudAppSecurityDocs: CloudAppSecurityDocs/conditional-access-app-control-how-to-overview.md

- [Active Directory Federation Services (AD FS) as your IdP](proxy-idp-adfs.md)
- [Okta as your IdP](proxy-idp-okta.md)

## Prerequisites: 

> 1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md)

Reviewer: ```suggestion
1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md).
```

> 1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](https://learn.microsoft.com/en-us/defender-cloud-apps/network-requirements)

Reviewer: ```suggestion
1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md)
```

> 1. Ensure your firewall configurations allow traffic from all the IP addresses listed [here](network-requirements.md).
> 1. Confirm that your application possesses a complete certificate chain. Incomplete or partial certificate chains may lead to unexpected behavior in applications when monitored with Conditional Access app control policies.

Reviewer 2: ```suggestion

1. Ensure your firewall configurations allow traffic from all the IP addresses listed on [Network requirements](network-requirements.md).
1. Confirm that your application possesses a complete certificate chain. Incomplete or partial certificate chains may lead to unexpected behavior in applications when monitored with Conditional Access app control policies.
```
